pkg/authorization/api/types.go

```diff
@@ -189,7 +189,7 @@ type ResourceAccessReview struct {
 	unversioned.TypeMeta
 
 	// Action describes the action being tested
-	Action AuthorizationAttributes
+	Action
```

Comment:
Name has to be the same as the type if on one side it's anonymous (which it
has to be to work correctly in JSON).  What I'll do is a follow up to make
the internal be Action Action.

On Fri, Jul 22, 2016 at 1:17 PM, David Eads <notifications@github.com>
wrote:

> In pkg/authorization/api/types.go
> <https://github.com/openshift/origin/pull/9814#discussion_r71913346>:
>
> > @@ -189,7 +189,7 @@ type ResourceAccessReview struct {
> >  	unversioned.TypeMeta
> >
> >  	// Action describes the action being tested
> > -	Action AuthorizationAttributes
> > +	Action
>
> anonymous inclusion makes me sad. We can't get autogeneration and keep
> this named?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/07c9f15316a1b3bd1893ee35004b6e2423917143..c983121be658b72bc96062df167bbb3a9160a1a6#r71913346>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pwg3pUbwSx6yAe1yBxdjQQuYP_Ewks5qYPsqgaJpZM4JLFc7>
> .
>